.circleci/config.yml

```diff
@@ -59,6 +59,7 @@ jobs:
               39) pyenv_py_ver="3.9.1" ;;
               310) pyenv_py_ver="3.10.1" ;;
               311) pyenv_py_ver="3.11.0" ;;
+              312) pyenv_py_ver="3.12.2" ;;
```

Comment:
We are able to release the pressure and ask only for minor version:
`3.7`, `3.8`, `3.9`, ...

Reply:
Oh, that's good to know! I thought they were specific because of reasons. I'll keep it in mind for the next time.